server/src/main/java/com/adobe/testing/s3mock/domain/FileStore.java

```diff
@@ -79,7 +79,7 @@
   private static final String META_FILE = "metadata";
   private static final String DATA_FILE = "fileData";
   private static final String PART_SUFFIX = ".part";
-  private static final String DEFAULT_CONTENT_TYPE = "application/octet-stream";
+  private static final String DEFAULT_CONTENT_TYPE = "binary/octet-stream";
```

Comment:
Is that the default that is used by AWS S3 as well?

Reply:
yep, e.g. [here](https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPUT.html)